wgs.md

```diff
@@ -0,0 +1,18 @@
+# Working Groups
```

Comment:
Can we expand wgs into a full name such as GROUPS.md (following the all uppercase convention for these top-level docs)